clouds/clouds-aws/src/test/resources/aws/cloudformation/eap.yaml

```diff
@@ -0,0 +1,186 @@
+        Type: "AWS::EC2::SecurityGroup"
+        Properties:
+            Tags:
+                -   Key: "cf-template"
+                    Value: !Ref defaultTag
+            GroupDescription: "Allows any connection"
+            GroupName: "allow-any"
+            VpcId: !Ref EC2VPC
+            SecurityGroupIngress:
+                -   CidrIp: "0.0.0.0/0"
+                    IpProtocol: "-1"
+                -   CidrIpv6: "::/0"
+                    IpProtocol: "-1"
+            SecurityGroupEgress:
+                -   CidrIp: "0.0.0.0/0"
+                    IpProtocol: "-1"
+
+    EC2InstanceEAP:
+        Type: "AWS::EC2::Instance"
+        CreationPolicy:
+            ResourceSignal:
+                Timeout: PT5M
+        DependsOn:
+            - EC2KeyPair
+        Properties:
+            ImageId: !Ref ImageID
+            InstanceType: "t3.medium"
+            KeyName: !Ref EC2KeyPair
+            AvailabilityZone: !GetAtt EC2Subnet.AvailabilityZone
+            Tenancy: "default"
+            SubnetId: !Ref EC2Subnet
+            EbsOptimized: false
+            SecurityGroupIds:
+                - !Ref EC2SecurityGroup
+            SourceDestCheck: true
+            IamInstanceProfile: EC2InstanceRole
+            BlockDeviceMappings:
+                -   DeviceName: "/dev/sda1"
+                    Ebs:
+                        Encrypted: false
+                        VolumeSize: 10
+                        VolumeType: "gp2"
+                        DeleteOnTermination: true
+            Tags:
+                -   Key: "cf-template"
+                    Value: !Ref defaultTag
+                -   Key: "cf-template-resource"
+                    Value: !Join [ "-", [!Ref defaultTag, "ec2instance"]]
+                -   Key: "Name"
+                    Value: !Ref instanceName
+            HibernationOptions:
+                Configured: false
+            CpuOptions:
+                CoreCount: 1
+                ThreadsPerCore: 2
+            EnclaveOptions:
+                Enabled: false
+            UserData:
+                Fn::Base64:
+                    !Sub |
+                        #!/usr/bin/env bash
+                        JBOSS_HOME=/opt/rh/eap7/root/usr/share/wildfly
+
+                        yum install -y unzip python3-pip
+                        pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz
+                        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+                        unzip -q awscliv2.zip
+                        chmod +x ./aws/dist/aws
+                        export PATH=$PATH:`pwd`/aws/dist
+
+                        INTERNAL_IP_ADDRESS=`ip addr show | grep eth0 -A 2 | head -n 3 | tail -n 1 | awk '{ print $2 }' | sed "s-/24--g" | cut -d'/' -f1`
+                        echo $INTERNAL_IP_ADDRESS > /root/internal-ip-address.txt
+                        echo "JAVA_OPTS=\"\$JAVA_OPTS \\
```

Comment:
This is just some testing hack? I as a customer can set such parameters externally to AMI image by e.g. env property?

Reply:
No. Customer needs to ssh into the VM and do this kind of steps. See https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/deploying_jboss_eap_on_amazon_web_services/reference_material#launching_on_amazon_ec2_using_script

Reply:
ok